assets/src/components/supported-templates-toggle/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { useContext } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Options } from '../options-context-provider';
+import { AMPSettingToggle } from '../amp-setting-toggle';
+
+export function SupportedTemplatesToggle() {
+	const { editedOptions, updateOptions } = useContext( Options );
+
+	const { all_templates_supported: allTemplatesSupported, theme_support: themeSupport } = editedOptions;
+
+	return [ 'standard', 'transitional' ].includes( themeSupport ) ? (
```

Comment:
This needs to change to also include `reader` _if_ the `reader_theme` is not `legacy`.

Actually, I suggest showing it _always_ but then making the toggle `disabled` if the `themeSupport === 'reader' && 'legacy === readerTheme`, and then show a notice to say:

> This setting is not available when the legacy Reader theme is selected.

This will then give incentive for someone to switch to the new mode.

Reply:
Updated in 1334179b7 
<img width="892" alt="Screen Shot 2020-07-09 at 1 28 59 PM" src="https://user-images.githubusercontent.com/9020968/87078587-7fde3b00-c1ea-11ea-9032-24b42ae13a1a.png">